BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -149,8 +149,11 @@ ErrorStatus BSP_CAN_Write(uint32_t id, uint8_t data[8], uint8_t length) {
     
     gTxMessage.StdId = id;
     gTxMessage.DLC = length;
-	for(int i = 0; i < length; i++){
-        gTxMessage.Data[i] = data[i];
+
+    if(length<=8){
```

Comment:
Is there a reason you chose 8? If so could you write it as a comment? There should be a reason there is no else statement following so if someone is looking at the code, they should know why we wouldn't send a CAN message if the size was less than 8. (Hint: if the reason is there is a max limit on CAN message sizes, leave it as a macro at the start of the file)